.github/workflows/package.yml

```diff
@@ -7,15 +7,13 @@ name: npm-build
 on:
   push:
     branches:
-      - main
       - release/**
     paths-ignore:
       - '.github/workflows/docusaurus.yml'
       - 'docs/**'
       - 'website/**'
   pull_request:
```

Comment:
This confused me as I was wondering why we are doing any publishing flow for PRs but further down, the actual publish job checks why we are running the action.  Not for this PR, but we should discuss refactoring this into a workflow that just does build and then include that into this publishing one to make the on triggers easier to understand.

Reply:
I'm no github workflow expert - I think Riko originally set this up. I beleive it runs on PRs and acts as a check that the build isn't broken, and yes the publish is inhibited on pull_request runs.

Now that you say that I think the second removal on line 18 is wrong as  it will stop it running on PRs so we lose that check.

It would indeed make sense to split these workflows up. For now I'll back out the change to the pull_request trigger.

Reply:
Fixed (but tightened up the PR event that it runs on so save FINOS some pennies)